### PR TITLE
[DOC-579] Docs for voyager v1.8.7

### PR DESCRIPTION
**Pull request by sudhanshu456**
@netlify /preview/yugabyte-voyager/release-notes/

### CODE REVIEW

File: docs/content/preview/yugabyte-voyager/release-notes.md [hunk 1]
```diff
@@ -13,6 +13,28 @@ type: docs
...
+## v1.8.7 - December 10, 2024
+
+### New Features
+
+- Introduced a framework in the `assess-migration` and `analyze-schema` commands to accept the target database version (`--target-db-version` flag) as input and use it for reporting issues not supported in that target version for the source schema.
+
+### Enhancements
+
+- Improved permission grant script (`yb-voyager-pg-grant-migration-permissions.sql`) by internally detecting table owners, eliminating the need to specify the `original_owner_of_tables` flag.
+- Enhanced reporting of "Unsupported Query Constructs" in the `assess-migration` command by filtering queries to include only those that match user-specified schemas, provided schema information is present in the query.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-voyager/release-notes.md:25]:
```suggestion
- Enhanced reporting of **Unsupported Query Constructs** in the `assess-migration` command by filtering queries to include only those that match user-specified schemas, provided schema information is present in the query.
```

File: docs/content/preview/yugabyte-voyager/release-notes.md [hunk 2]
```diff
@@ -13,6 +13,28 @@ type: docs
...
+- Enhanced the `assess-migration` and `analyze-schema` commands to report issues in Functions or Procedures for variables declared with reference types (%TYPE) under the Unsupported PL/pgSQL Objects section.
+- Added support to report DDL issues present in the PL/pgSQL blocks of objects listed in the "Unsupported PL/pgSQL Objects" section of the `assess-migration` and `analyze-schema` commands.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-voyager/release-notes.md:27]:
```suggestion
- Added support to report DDL issues present in the PL/pgSQL blocks of objects listed in the **Unsupported PL/pgSQL Objects** section of the `assess-migration` and `analyze-schema` commands.
```

File: docs/content/preview/yugabyte-voyager/release-notes.md [hunk 3]
```diff
@@ -13,6 +13,28 @@ type: docs
...
+- Allow yb-voyager upgrades during migration from the recent breaking release (`1.8.5`) to the higher versions in the same migration.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-voyager/release-notes.md:28]:
```suggestion
- Allow yb-voyager upgrades during migration from the recent breaking release (v1.8.5) to later versions.
```

File: docs/content/preview/yugabyte-voyager/release-notes.md [hunk 4]
```diff
@@ -13,6 +13,28 @@ type: docs
...
+
+### New Features
+
+- Introduced a framework in the `assess-migration` and `analyze-schema` commands to accept the target database version (`--target-db-version` flag) as input and use it for reporting issues not supported in that target version for the source schema.
+
+### Enhancements
+
+- Improved permission grant script (`yb-voyager-pg-grant-migration-permissions.sql`) by internally detecting table owners, eliminating the need to specify the `original_owner_of_tables` flag.
+- Enhanced reporting of "Unsupported Query Constructs" in the `assess-migration` command by filtering queries to include only those that match user-specified schemas, provided schema information is present in the query.
+- Enhanced the `assess-migration` and `analyze-schema` commands to report issues in Functions or Procedures for variables declared with reference types (%TYPE) under the Unsupported PL/pgSQL Objects section.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-voyager/release-notes.md:26]:
```suggestion
- Enhanced the `assess-migration` and `analyze-schema` commands to report issues in Functions or Procedures for variables declared with reference types (%TYPE) in the **Unsupported PL/pgSQL Objects** section.
```

File: docs/content/preview/yugabyte-voyager/release-notes.md [hunk 5]
```diff
@@ -13,6 +13,28 @@ type: docs
...
+- Added support to report DDL issues present in the PL/pgSQL blocks of objects listed in the "Unsupported PL/pgSQL Objects" section of the `assess-migration` and `analyze-schema` commands.
+- Allow yb-voyager upgrades during migration from the recent breaking release (`1.8.5`) to the higher versions in the same migration.
+- Modified the internal HTTP port to dynamically use an available free port instead of defaulting to 8080, avoiding conflicts with commonly used services.
+- Added a guardrail check to the `assess-migration` command to verify that the `pg_stat_statements` extension is properly loaded in the source database.
+
+### Bug fixes
+
+- Fixed an [issue](https://github.com/yugabyte/yb-voyager/issues/1895) where `NOT VALID` constraints in the schema could cause constraint violation errors during data imports; these constraints are now created during the post-snapshot import phase.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-voyager/release-notes.md:34]:
```suggestion
- Fixed an [issue](https://github.com/yugabyte/yb-voyager/issues/1895) where NOT VALID constraints in the schema could cause constraint violation errors during data imports; these constraints are now created during the post-snapshot import phase.
```

File: docs/content/preview/yugabyte-voyager/release-notes.md [hunk 6]
```diff
@@ -13,6 +13,28 @@ type: docs
...
+- Fixed formatting issues in the assessment HTML report, where extra spaces or characters appeared after the "Unsupported PL/pgSQL Objects" section heading, depending on the browser used for viewing.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-voyager/release-notes.md:35]:
```suggestion
- Fixed formatting issues in the assessment HTML report, where extra spaces or characters appeared after the **Unsupported PL/pgSQL Objects** heading, depending on the browser used for viewing.
```

**Review by ddhodge — APPROVED** [review]
misc edits

File: docs/content/preview/yugabyte-voyager/migrate/live-migrate.md [hunk 7]
```diff
@@ -421,13 +420,12 @@ You can use only one of the following arguments to connect to your Oracle instan
...
         -d <database> \
-        -U <username> \
+        -U <username> \ # A superuser or a privileged user with enough permissions to grant privileges
         -v voyager_user='ybvoyager' \
         -v schema_list='<comma_separated_schema_list>' \
         -v is_live_migration=1 \
         -v is_live_migration_fall_back=0 \
         -v replication_group='<replication_group>' \
-        -v original_owner_of_tables='<original_owner_of_tables>' \
         -f <path_to_the_script>
```
Comment (priyanshi-yb) [inline docs/content/preview/yugabyte-voyager/migrate/live-migrate.md:429]: Please remove this flag `original_owner....` in all the live migration workflows live-fall-forward, live -fallback.

**Review by priyanshi-yb — APPROVED** [review]
LGTM

File: docs/content/preview/yugabyte-voyager/migrate/live-migrate.md (hunk 7 above)
Comment (sudhanshu456) [reply docs/content/preview/yugabyte-voyager/migrate/live-migrate.md:429]: Done

**Pull request merged**